src/applications/deploying-apps.md

```diff
@@ -100,6 +100,14 @@ Where you will see differences while using the CLI is `deis ps:list` will output
 Behind the scenes your application deploy will be built up of a Deployment object per process type,
 each having multiple ReplicaSets (one per release) which in turn manage the Pods running your application.
 
+!!! Known Issue
+		If `DEIS_KUBERNETES_DEPLOYMENTS` is enabled and there is a deployment in progress when the
```

Comment:
Do you mean an extra newline below `!!! Known Issue`? Otherwise, I want the entire text to be inside the warning box